chapters/en/slides/module6/module6_01.md

```diff
@@ -2,242 +2,319 @@
+but not on Windows.
+
+When using delimiters to separate metadata fields,
+it is important to be consistent across files with similar content
+as we can see an example of in the screenshot in this slide.
+
+Deliberate and consistent use of delimiters
+make it easy to recover all metadata in our filenames in the future,
+using pattern matching by globbing or regular expressions
+as we will see in the next slide.
 
-Although file names should never contain punctuation characters such as `*` or `?` (because they have special meaning in e.g. globbing and regular expressions), we can use underscores or hyphens to separate units of meta data or words in a file name.
+---
 
-This makes it easy to recover all meta data in our file names in the future, using pattern matching by globbing or regular expressions.
+## 2. Machine-readable filenames: Pattern matching in the terminal
 
----
+Files in the directory:
 
-## Recap
+<img src="/module6/plasmid_names.png" width="50%"></img>
 
 <br>
 
-Machine-readable file names are:
+**Example:** Globbing with <code>ls</code>to narrow file listing:
 
-- Easy to search for files later
-- Easy to narrow file lists based on names
-- Easy to extract info from file names, e.g. by splitting
+<img src="/module6/plasmid_glob.png" width="60%"></img>
 
-<br>
+Notes:
+Machine-readable filenames make it easy to find a group of desired filenames by using special patterns to match parts of the filenames.
+These patterns are called "glob"s, and finding filenames using this method is called "globbing".
 
-Even if you're new to regular expressions and globbing, be kind to yourself and avoid
-- Spaces in file names
-- Punctuation
-- Accented characters
-- Different files named `foo` and `Foo`
+In this example here, we have specified a particular pattern, namely `*Plasmid*` to find all filenames that contain this word. This is an example of "globbing" using the `*` wildcard as you might remember from module 2.
 
-Notes:
+If we've already been careful to include `Plasmid` in the name of all files with a related content, then finding those files using a script or in the terminal would be very easy.
 
 ---
 
-## 2. Human-readable file names
+## 2. Machine-readable filenames: Pattern matching in the file browser
+
+**Example:** Using MacOS Finder search
 
 <br>
 
-- Name contains info on content
-- Connects to the concept of a _slug_ from semantic URLs
+<img src="/module6/plasmid_mac_os_search.png" width="70%"></img>
 
-<br>
+Notes:
 
-**Example:**
+Using these filename conventions are not only useful in the terminal.
+They also makes it easy to find the files you need in file browser on Linux, MacOS, or Windows,
+as can be seen in this slide.
 
-Which set of file names do you want to see at 3 a.m. before a deadline?
+---
 
-<img src="/module6/human_readable_not_options.png" width="80%"></img>
+## 2. Machine-readable filenames: Pattern matching in a programming language
 
-Notes:
+**Example:** Using regex in Python
```

Comment:
@armanawn Could we change this example to use Python instead of R since that's the language used in all other courses in this Data Science specialization? You could just change the input code and crop the output screenshot to include the R portion if that is easier?